soundcard/pulseaudio.py

```diff
@@ -333,26 +332,17 @@ class _Recorder(_Stream):
 
     """
 
+    def __init__(self, id, samplerate, channels, blocksize=None,
+                 name='outputstream', fix_blocksize=False):
```

Comment:
We don't need `fix_blocksize` any longer

Reply:
And it might be cleaner to capture all arguments as `self, *args, **kwargs`, if you want.

Reply:
I didn't pay attention to `fix_blocksize`, yep.
You're right about `self, *args, **kwargs`